server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    sitter_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    start_date: {
+      type: Date,
+      required: true,
+    },
+    end_date: {
+      type: Date,
+      required: true,
+    },
```

Comment:
Also, it would be nice to see a sort of validation here that ensures `end > start` 😎

Reply:
Don't forget to camelCase our property keys 😄

Reply:
Added validation to endDate.